compressor/contrib/jinja2ext.py

```diff
@@ -45,11 +45,10 @@ def _compress_normal(self, kind, mode, caller):
 
     def _compress(self, kind, mode, caller, forced):
         mode = mode or OUTPUT_FILE
-        original_content = caller()
         context = {
-            'original_content': original_content
+            'original_content_generator': caller,
         }
         return self.render_compressed(context, kind, mode, forced=forced)
 
     def get_original_content(self, context):
-        return context['original_content']
+        return context['original_content_generator']()
```

Comment:
I wanted to ask you about this.

In Django you always re-render the node contents but in Jinja you render it once and then re-use. This has brought some problems when I was overriding Django settings. Now it doesn't, but shouldn't it be implemented in the same way for Django and Jinja?